ra-optimisation/search-tree-rao/src/test/java/com/powsybl/openrao/searchtreerao/searchtree/algorithms/LeafTest.java

```diff
@@ -197,6 +205,26 @@ void testMultipleLeafDefinitionWithSameNetworkAction() {
         assertFalse(leaf2.isRoot());
     }
 
+    @Test
+    void testLeafDefinitionWithAcEmulationDeactivationNetworkAction() {
```

Comment:
could you add a comment explaining (once more) that what we're testing is just that in Leaf definition we're updating the hvdcLine's active power setpoint in the network with what was updated in the loadflow : P in terminals.